idaes/core/surrogate/pysmo/sampling.py

```diff
@@ -1412,6 +1422,12 @@ def __init__(
             raise Exception("Invalid tolerance input")
         self.eps = tolerance
 
+        if rand_seed is not None:
```

Comment:
Seeing as this is in `__init__`, should this code be moved to the base class to avoid duplicating the code in each subclass?

Reply:
No, I'd rather leave it here because it is an argument specific to a subset of the sampling methods.